ivadomed/loader/utils.py

```diff
@@ -254,7 +255,7 @@ def imed_collate(batch):
         if elem.shape == ():  # scalars
             py_type = float if elem.dtype.name.startswith('float') else int
             return __numpy_type_map[elem.dtype.name](list(map(py_type, batch)))
-    elif isinstance(batch[0], int_classes):
+    elif isinstance(batch[0], int):
```

Comment:
This is for the Torch 1.9+ fix right? I.e. if we leave the original import `int_classes`, it will break on Torch 1.9+?

Reply:
Yes, that is right.